piet-scene/src/geometry.rs

```diff
@@ -0,0 +1,225 @@
+// Copyright 2022 The piet-gpu authors.
```

Comment:
Some of this code seems pretty familiar. That's ok, but maybe it should say something like "based in part on kurbo."

Similarly, a pretty common pattern would be to have kurbo as an optional dependency, and have conversions in place. I'm not sure that should be done this round, perhaps in future. I think the Rust traits for approximate conversions (f64->f32 and friends) were still a bit controversial last time I took a serious look at this, not sure if that's been settled by now.

Reply:
I did pull most of the Affine functions from kurbo. I'll add an attribution. 

Happy to prep another PR adding the optional dependency along with the conversions and changing some signatures to take `impl Into<T>` for the appropriate types.